### PR TITLE
feat(contribution-types): support plural forms, multi word types and any case

### PR DESCRIPTION
**Pull request by jakebolam**
Implements #15.

Alternative approach to #37

### CODE REVIEW

File: src/utils/parse-comment/index.js [hunk 1]
```diff
@@ -22,16 +23,58 @@ const validContributionTypes = [
...
 ]
 
+// Types that are valid multi words, that we need to re map back to there camelCase parts
+const validMultiContributionTypesMapping = {
+    eventorganizing: 'eventOrganizing',
+    fundingfinding: 'fundingFinding',
+    usertesting: 'userTesting',
+}
+
+// Additional terms to match to types (plurals, aliases etc)
```
Comment (jakebolam) [inline src/utils/parse-comment/index.js]: I investigated using compromise for plurals and it fell quite shot (got 15 of our needed 21) also missed some plural forms.

**Comment by Berkmann18** [comment]
I like that 😄 !
What do you guys think about [this approach](https://github.com/all-contributors/all-contributors-bot/pull/37/commits/9e94dd002cccd2e5bdc687a0a093d02226916263)?

**Comment by jakebolam** [comment]
@Berkmann18 [replied](https://github.com/all-contributors/all-contributors-bot/pull/37/commits/9e94dd002cccd2e5bdc687a0a093d02226916263#r248626539)

**Comment by jakebolam** [comment]
@Berkmann18 you good if I merge this?

**Pull request merged**